core/src/main/java/org/n52/iceland/request/AbstractServiceRequest.java

```diff
@@ -175,6 +180,18 @@ public Locale getRequestedLocale() {
         return LocaleHelper.fromString(getRequestedLanguage());
     }
 
+    public String getOriginalRequest() {
```

Comment:
I hereby proclaim the joyful news that the great Huddle has come to a decision: **expose the `Optional`s!**